db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "The internal database ID."
+    change_column_comment :miq_tasks, :identifier, "The resource this task is related to in '#{klass}:#{id}' format."
+    change_column_comment :miq_tasks, :message, "Message that describes the reason for the current status."
+    change_column_comment :miq_tasks, :miq_server_id, "The ID of the ManageIQ server where the task was executed."
+    change_column_comment :miq_tasks, :name, "The name of the task. Typically doubles as a description."
+    change_column_comment :miq_tasks, :pct_complete, "The current completion percentage of the task."
+    change_column_comment :miq_tasks, :results, "TODO"
+    change_column_comment :miq_tasks, :started_on, "The timestamp the task was actually started."
+    change_column_comment :miq_tasks, :state, "The current state of the task - Initialized, Queued, Active, or Finished."
```

Comment:
There are more possible states and other states could be added later on (vs status with has more-less frozen possible values). 
![screen shot 2019-02-14 at 12 01 47 pm](https://user-images.githubusercontent.com/6556758/52804129-6e265180-3051-11e9-9b50-415ec6ae82c2.png)

Reply:
Hm, I was looking at miq_task.rb. Those must be added in subclasses somewhere.

Reply:
Ok, updated so that those are just provided as examples.